tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -231,6 +231,50 @@ public async Task GetOrderSummaryAsync_ServiceRecipientCount_ReturnsCountOfTwo()
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [TestCase]
+        public async Task GetOrderSummaryAsync_ServiceCatalogueSolutionCount_ReturnsCountOfTwo()
+        {
+            var order = OrderBuilder.Create()
+                .WithOrderItem(OrderItemBuilder.Create()
+                    .WithCatalogueItemType(CatalogueItemType.Solution)
+                    .Build())
+                .WithOrderItem(OrderItemBuilder.Create()
+                    .WithCatalogueItemType(CatalogueItemType.Solution)
+                    .Build()
+                ).Build();
+
+            var context = OrdersControllerTestContext.Setup(order.OrganisationId);
+            context.Order = order;
+            
+            var controller = context.OrdersController;
```

Comment:
Can you integrate the controller into the next line, so instead of controller. just do context.OrdersController ect